assets/chef-recipes/cookbooks/mariadb/recipes/mdberepos.rb

```diff
@@ -94,12 +94,23 @@
     action :update
   end
 when 'rhel', 'fedora', 'centos', 'almalinux'
-  yum_repository repo_file_name do
-    description 'MariaDB Enterprise Server'
-    baseurl node['mariadb']['repo']
-    gpgkey node['mariadb']['repo_key']
-    sensitive true
-    options({ 'module_hotfixes' => '1' })
+  if node['mariadb']['repo'].include?('cspkg')
```

Comment:
Repository information must include property whether to enable or disable GPG check.